test/unit/plugin/retryFailedStep_test.js

```diff
@@ -0,0 +1,74 @@
+const retryFailedStep = require('../../../lib/plugin/retryFailedStep');
+const sinon = require('sinon');
```

Comment:
not really needed here

Reply:
Ah, sorry, I forgot to delete it :bowing_man: